src/components/dialogmote/DialogmoteInnkallingContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { useParams } from "react-router-dom";
+import Side from "../../sider/Side";
+import { MOETEPLANLEGGER } from "../../enums/menypunkter";
+import React, { ReactElement } from "react";
+import Sidetopp from "../Sidetopp";
+import DialogmoteInnkallingSkjema from "./DialogmoteInnkallingSkjema";
+import { useAppSelector } from "../../hooks/hooks";
+import LasterSide from "../LasterSide";
+import styled from "styled-components";
+import AlertStripe from "nav-frontend-alertstriper";
+
+const texts = {
```

Comment:
Smak og behag kanskje, men foretrekker å inline tekster når de ikke brukes ulike steder

Reply:
Ja, prøvde å følge mønsteret fra ellers i appen, men for så vidt enig så kan godt endre 👍

Reply:
Det er et bevisst valg å skille tekster fra visning. Av to grunner: Det er da lettere for ikke-tekniske å endre ledeteskter. I tillegg er det nyttig å skille teskter fra visning, av samme grunn som man ønsker å skille styling fra visning, logikk fra visning osv.

Reply:
Jeg er enig i det gitt at man skal støtte flere språk - men da skiller man det typisk ut i egne filer.

Vi får ta en battle på det litt senere 💥 🤛 👊 🤜 💥

Reply:
Lar det være slik så det er konsistent med resten av appen(e) (inntil det er battlet over 👊 )

Reply:
👍